src/app/[locale]/topics/components/common/LearningSpace.tsx

```
@@ -86,42 +88,17 @@ export default function LearningSpace(props: Props) {
-                                    </TabsTrigger>
-                                    <TabsTrigger
-                                        value="classwork"
-                                        className="w-[30%] data-[state=active]:bg-background/80 data-[state=active]:shadow-sm 
-                                                data-[state=active]:border-b-2 data-[state=active]:border-primary 
-                                                rounded-b-none text-base font-medium px-4 py-2 transition-all 
-                                                -mb-px"
-                                    >
-                                        Classwork
-                                    </TabsTrigger>
+                                                -mb-px`}
+                                        >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)